lisa/tools/mkfs.py

```diff
@@ -31,16 +38,60 @@ def mkfs(self, disk: str, command: str) -> None:
             )
         cmd_result.assert_exit_code()
 
+    def format_disk(self, disk_name: str, file_system: FileSystem) -> None:
+        if file_system == FileSystem.xfs:
+            mkfs_xfs = self.node.tools[Mkfsxfs]
+            mkfs_xfs.mkfs(disk_name, f"{file_system} -f")
```

Comment:
Does the -f is necessary for this type only? If not below line can be shared for all tools

`mkfs_xfs.mkfs(disk_name, f"{file_system} -f")`